### PR TITLE
Adding kubebuilder and controller-runtime meeting to api machinery sig

### PR DESCRIPTION
**Pull request by shawn-hurley**
/cc @pwittrock @DirectXMan12 @droot 

Adding a scheduled meeting time for the controller-runtime and kubebuilder projects to meet and discuss issues and features.

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -25,6 +25,11 @@ sigs:
       url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
       archive_url: https://goo.gl/0lbiM9
       recordings_url: https://www.youtube.com/watch?v=Lj1ScbXpnpY&list=PL69nYSiGNLP21oW3hbLyjjj4XhrwKxH2R
+    - description: Kubebuilder and Controller Runtime Meeting
+      day: Thursday
+      time: "10:00"
+      tz: "PT (Pacific Time)"
+      frequency: monthly - third Wednesday every month
```
Comment (mhrivnak) [inline sigs.yaml]: Wed or Thu?

File: sig-api-machinery/README.md [hunk 2]
```diff
@@ -14,6 +14,7 @@ Covers all aspects of API server, API registration and discovery, generic API CR
 * Regular SIG Meeting: [Wednesdays at 11:00 PT (Pacific Time)](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=11:00&tz=PT%20%28Pacific%20Time%29).
   * [Meeting notes and Agenda](https://goo.gl/0lbiM9).
   * [Meeting recordings](https://www.youtube.com/watch?v=Lj1ScbXpnpY&list=PL69nYSiGNLP21oW3hbLyjjj4XhrwKxH2R).
+* Kubebuilder and Controller Runtime Meeting: [Thursdays at 10:00 PT (Pacific Time)]() (monthly - third Wednesday every month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:00&tz=PT%20%28Pacific%20Time%29).
```
Comment (mhrivnak) [inline sig-api-machinery/README.md]: Wed or Thu?

File: sig-list.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ When the need arises, a [new SIG can be created](sig-creation-procedure.md)
...
 |------|-------|--------|---------|----------|
-|[API Machinery](sig-api-machinery/README.md)|api-machinery|* [Daniel Smith](https://github.com/lavalamp), Google<br>* [David Eads](https://github.com/deads2k), Red Hat<br>|* [Slack](https://kubernetes.slack.com/messages/sig-api-machinery)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-api-machinery)|* Regular SIG Meeting: [Wednesdays at 11:00 PT (Pacific Time) (biweekly)](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit)<br>
+|[API Machinery](sig-api-machinery/README.md)|api-machinery|* [Daniel Smith](https://github.com/lavalamp), Google<br>* [David Eads](https://github.com/deads2k), Red Hat<br>|* [Slack](https://kubernetes.slack.com/messages/sig-api-machinery)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-api-machinery)|* Regular SIG Meeting: [Wednesdays at 11:00 PT (Pacific Time) (biweekly)](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit)<br>* Kubebuilder and Controller Runtime Meeting: [Thursdays at 10:00 PT (Pacific Time) (monthly - third Wednesday every month)]()<br>
```
Comment (mhrivnak) [inline sig-list.md]: Wed or Thu?

**Comment by DirectXMan12** [comment]
/hold

this conflicts with the community meeting right now.  We should move it to another time.

**Comment by shawn-hurley** [comment]
@DirectXMan12 Can you double check the date now?

**Comment by DirectXMan12** [comment]
it still says Thursday at 10:00 PT, which is community meeting time

**Comment by shawn-hurley** [comment]
@cblecker @DirectXMan12 

Is there more to be done to merge this PR?

**Comment by DirectXMan12** [comment]
I think it was just stuck on the hold

**Pull request merged**